frontend/packages/console-dynamic-plugin-sdk/src/api/internal-types.ts

```diff
@@ -249,3 +249,40 @@ export enum ActionMenuVariant {
   KEBAB = 'plain',
   DROPDOWN = 'default',
 }
+
+export type DroppableEditYAMLProps = {
+  allowMultiple?: boolean;
+  obj: string;
+};
+
+export type HandlePromiseProps = {
+  handlePromise: <T>(
+    promise: Promise<T>,
+    onFulfill?: (res) => void,
+    onError?: (errorMsg: string) => void,
+  ) => void;
+  inProgress: boolean;
+  errorMessage: string;
+};
+
+export type K8sResourceKind = K8sResourceCommon & {
```

Comment:
This one is already declared in `packages/console-dynamic-plugin-sdk/src/api/internal-api-kubevirt.ts` but we plan to remove that SDK package in #11091

It's also declared in `packages/console-dynamic-plugin-sdk/src/app/k8s/actions/k8s.ts` :smiley: 

So I suggest to move `K8sResourceKind` from `packages/console-dynamic-plugin-sdk/src/app/k8s/actions/k8s.ts` to `packages/console-dynamic-plugin-sdk/src/api/common-types.ts` and use it also here, in order to avoid unnecessary type duplicity.

Reply:
hi, i moved it to extensions/common-types, otherwise im getting circular dependency :(